labelbox/schema/project.py

```diff
@@ -352,6 +352,47 @@ def create_prediction(self, label, data_row, prediction_model=None):
         res = self.client.execute(query_str, params)
         return Prediction(self.client, res["createPrediction"])
 
+    def upload_annotations(
+            self,
+            name: str,
+            url: str = None,
+            path: str = None,
+            annotations: Iterable[dict] = None
+    ) -> None:
```

Comment:
`-> BulkImportRequest` - we will have a circular dependency though...

Reply:
the way to resolve this with mypy is to just do the string version `'BulkImportRequest'`